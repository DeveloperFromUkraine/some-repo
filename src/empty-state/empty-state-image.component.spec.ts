import { ComponentFixture, TestBed } from '@angular/core/testing';
import { EmptyStateImageComponent } from '../index';
import { ComponentTest } from '../../test/test-bed/component';
import { Component } from '@angular/core';

describe('Empty State Image', () => {
  let fixture: ComponentFixture<EmptyStateImageComponent>;

  beforeEach(() => {
    ComponentTest.createTestBed([], [EmptyStateImageComponent] as Component[]);

    fixture = TestBed.createComponent(EmptyStateImageComponent);
  });

  it('should match snapshot', () => {
    expect(fixture).toMatchSnapshot();
  });
});
