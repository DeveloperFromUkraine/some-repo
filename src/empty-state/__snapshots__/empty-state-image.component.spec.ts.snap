// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyStateImageComponent should match snapshot 1`] = `
<ign-empty-state-image>
  <img
    _ngcontent-c0=""
    class="image"
    src="/assets/images/empty-state.svg"
    width="640px"
  />'
</ign-empty-state-image>
`;
